src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlJsonDbFunctionsTranslator.cs

```diff
@@ -96,7 +96,13 @@ public class NpgsqlJsonDbFunctionsTranslator : IMethodCallTranslator
                 => _sqlExpressionFactory.MakePostgresBinary(PostgresExpressionType.JsonExistsAny, Jsonb(args[0]), args[1]),
             nameof(NpgsqlJsonDbFunctionsExtensions.JsonExistAll)
                 => _sqlExpressionFactory.MakePostgresBinary(PostgresExpressionType.JsonExistsAll, Jsonb(args[0]), args[1]),
-
+            nameof(NpgsqlJsonDbFunctionsExtensions.JsonPathExists)
+                => _sqlExpressionFactory.Function(
+                    "jsonb_path_exists",
+                    new[] { args[0], args[1] },
+                    nullable: false,
+                    argumentsPropagateNullability: FalseArrays[2],
```

Comment:
jsonb_path_exists does propagate NULL, i.e. passing null to the 1st two arguments causes the function to return null, like most functions in SQL:

```sql
SELECT jsonb_path_exists(NULL, '$.a[1]'); -- NULL
SELECT jsonb_path_exists('{"a":[1,2,3,4,5]}', NULL); --NULL
```

Assuming you add support for the two additional parameters (vars, silent), can you please confirm the behavior for these?

```suggestion
                    argumentsPropagateNullability: TrueArrays[2],
```

Reply:
Thanks, I wasn't really sure what exactly that was doing and just took a guess. The additional parameters also propagate NULL so I adjusted as needed.